class.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'module'
+class Mylist
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
Inside Mylist class you should implement #each method that yields all members of @list in succession as mentioned in the project requirements , Kindly fix this issue 👍

Reply:
THANKS @Danieldotcomcoder